apps/explorer/src/pages/object-result/views/TokenView.tsx

```diff
@@ -202,7 +203,21 @@ function TokenView({ data }: { data: DataType }) {
                             {properties.map(([key, value], index) => (
                                 <tr key={index}>
                                     <td>{key}</td>
-                                    <td>{value}</td>
+                                    <td>
+                                        {/* TODO: Use normalized module to determine this display. */}
+                                        {value.startsWith('http://') ||
```

Comment:
Seems e2e caught a potential error! probably we need `typeof value === 'string' &&` as well